packages/web/src/common/store/tipping/sagas.ts

```diff
@@ -321,13 +322,39 @@ function* sendTipAsync() {
 
   const senderUserId = encodeHashId(sender.user_id)
   const receiverUserId = encodeHashId(receiver.user_id)
+
   const amount = Number(stringAudioAmount)
 
+  let senderWallet = '' as SolanaWalletAddress
```

Comment:
icky. can we instead do

```
let senderWallet

try {
  ...
} catch (e) {
  ...
}

yield put(
      make(Name.TIP_AUDIO_REQUEST, {
        senderWallet: senderWallet ?? '',
        recipientWallet: recipientWallet ?? '',
        senderHandle: sender.handle,
        recipientHandle: receiver.handle,
        amount,
})

Reply:
Oh yep, definitely.

Reply:
Decided instead to make the values optional in the analytics payload and update the type at declaration to be `SolanaWalletAddress | undefined` to make the compiler happy about the value being used before assignment.